src/webgpu/api/validation/encoding/cmds/render/dynamic_state.spec.ts

```diff
@@ -296,7 +297,7 @@ g.test('setBlendColor')
     const { r, g, b, a } = t.params;
     const encoders = t.createDummyRenderPassEncoder();
     /* eslint-disable-next-line deprecation/deprecation */
```

Comment:
Same here. Can the comment be removed now that we're not using the deprecated function?